### DISCUSSION TITLE
Change default height

### DISCUSSION THREAD

**Discussion by Wulfheart** [awcodes/filament-tiptap-editor#188] [Q&A] [answered]
Hello,

at first, thank you for creating this package. I especially like the integration with curator.

I created it in a form and it looks like this:
![image](https://github.com/awcodes/filament-tiptap-editor/assets/25671390/8116a2a0-2b00-49a7-97dd-0b8ac5f5abf4)

I would like to make it a little bit higher by default and I cannot find a method to achieve this. The `->height()` does not seem to work according to my autocompletion. 

My current element code looks like this:
```php
public static function form(Form $form): Form
    {
        return $form
            ->schema([
                TiptapEditor::make('content')->columnSpanFull(),
                //
            ]);
    }
```

**Comment by awcodes** [comment, marked as answer]
It's in the README. 😀

[https://github.com/awcodes/filament-tiptap-editor#initial-height-of-editor-field](https://github.com/awcodes/filament-tiptap-editor#initial-height-of-editor-field)

**Reply by Wulfheart** [reply]
I am so sorry. Thank you very much.
